builder/qemu/step_run.go

```diff
@@ -190,14 +190,26 @@ func getCommandArgs(bootDrive string, state multistep.StateBag) ([]string, error
 		}
 	}
 
+	cdPaths := []string{}
+	// Add the installation CD to the run command
 	if !config.DiskImage {
+		cdPaths = append(cdPaths, isoPath)
+	}
+	// Add our custom CD created from cd_files, if it exists
+	cdFilesPath, ok := state.Get("cd_path").(string)
+	if ok {
+		if cdFilesPath != "" {
+			cdPaths = append(cdPaths, cdFilesPath)
+		}
+	}
```

Comment:
NIT:

```suggestion
	cdFilesPath, _ := state.Get("cd_path").(string)
	if cdFilesPath != "" {
		cdPaths = append(cdPaths, cdFilesPath)
	}
```

Reply:
Using state.Get with a type assertion at the same time will cause a nil interface conversion crash if cd_files isn't set, and you can't do the type assertion like that with GetOk becuase it returns multiple values.

Reply:
The suggestion didn't add a type assertion, but instead ignored the boolean returned by the type assertion, just to make things simpler.

so from:
```go
cdFilesPath, isAString := state.Get("cd_path").(string)
```
we go to
```go
cdFilesPath, _ := state.Get("cd_path").(string)
```
in that case cdFilesPath will always be a string, sometimes empty, ex: https://play.golang.org/p/EL4DxX1suii